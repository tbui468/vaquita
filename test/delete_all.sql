if exists drop database sol;
create database sol;
open sol;

create table planets (
                        id int key,
                        name string, 
                        mass float, 
                        number_of_moons int,
                        surface_pressure float,
                        ring_system bool
                    );

insert into planets (
                        id,
                        name, 
                        mass, 
                        number_of_moons, 
                        surface_pressure,
                        ring_system
                    ) 

values
        (1, "Venus", 4.87, 0, 92.0, false),
        (2, "Earth", 5.97, 1, 1.0, false),
        (3, "Mars", 0.642, 2, 0.01, false);

delete from planets;
select * from planets;

close sol;
drop database sol;
exit;
