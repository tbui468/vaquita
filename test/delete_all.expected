created database sol
opened database sol
created table planets
inserted 3 record(s) into planets
closed database sol
dropped database sol
